sympy/polys/densetools.py

```diff
@@ -1199,13 +1199,18 @@ def dup_clear_denoms(f, K0, K1=None, convert=False):
     for c in f:
         common = K1.lcm(common, K0.denom(c))
 
-    if not K1.is_one(common):
-        f = dup_mul_ground(f, common, K0)
+    if K1.is_one(common):
+        if not convert:
+            return common, f
+        else:
+            return common, dup_convert(f, K0, K1)
+
+    f = [K0.numer(c)*K1.quo(common, K0.denom(c)) for c in f]
```

Comment:
Please include the comment that I showed in the example code:
```
# Use quo rather than exquo to handle inexact domains by discarding the
# remainder.
```

Reply:
Done sir